internal/gapicgen/cmd/genbot/github.go

```diff
@@ -76,11 +77,13 @@ type PullRequest struct {
 	IsOpen  bool
 	Number  int
 	Repo    string
+	IsDraft bool
 }
 
 // GithubClient is a convenience wrapper around a github.Client.
```

Comment:
This comment is slightly out of date, as GithubClient is now a wrapper around two github clients.

Reply:
Done.